main.cpp

```diff
@@ -126,26 +126,26 @@ void displayDigit(uint8_t digit, uint8_t position)
     //timer.reset();
 }
 
-void displayAccelerating(~) // name of function to indicate accelerating
+void displayAccelerating(digitalRead(mbed[25])) // function to indicate accelerating
 { 
-    if (mbed[25] = 1) //mbed[25] is the pin number on LPC1768-mbed for ACCEL_OUT. As accelerator is a switch, only on or off so signal is 1 or 0. Not sure if this is the right way to find out what position the accelerator switch is in
-    {
+
+    if (digitalRead(mbed[25]) = 1) { //mbed[25] is the pin number on LPC1768-mbed for ACCEL_OUT. As accelerator is a switch, only on or off so signal is 1 or 0. Not sure if this is the right way to find out what position the accelerator switch is in
```

Comment:
Would you mind double checking this? think it should be `==` to do an equality check. 
https://www.w3schools.com/cpp/cpp_operators_comparison.asp

Reply:
I think you're right Adam.  Thanks for pointing them out.